404.md

```diff
@@ -1,17 +1,11 @@
 ---
 permalink: /404.html
-layout: error
-errormessage: 🙅‍♀️ 404
-body_class: homepage
-errorhuman: "OpenSearch can't find that page." 
+title: 404
+layout: default
+heading_anchors: false
+nav_exclude: true
 ---
 
-Sorry about that. Perhaps we moved something around, or you mistyped the URL? 
-
-
-Head to the [OpenSearch Documentation home page](https://opensearch.org/docs/latest/) or try our [community forum](https://forum.opensearch.org/).
-
-
-
-
+# OpenSearch cannot find that page. 
 
+Perhaps we moved something around, or you mistyped the URL? Try using search or go to the [OpenSearch Documentation home page](https://opensearch.org/docs/latest/). If you need further help, see the [OpenSearch community forum](https://forum.opensearch.org/).
```

Comment:
No comma in the first sentence. In how many instances would a user see this message because they mistyped a URL? If not many or none, then I would try not to blame the user and just state that the content has been moved. In general, it's not a great user experience to provide an error with a question in response to what will presumably most often be the result of something we did, in which case we should be redirecting the user. This statement reads as something of a shrug. Why do we want the user to go to the home page, and what do we want them to do once there? Further, why would the community forum be an avenue of support after receiving this error?